.travis.yml

```diff
@@ -6,12 +6,18 @@ android:
     - tools
     - build-tools-28.0.3
     - android-28
+    - android-30
```

Comment:
I would suggest you to use some lower level of Emulator as it may lead to compatibility issues

Reply:
I installed Pixel 3a emulator android-28, that would do.

Reply:
@gaurivn this has nothing to do with your the emulator on your system. Here you are just specifying emulator specifications that would run on Docker on Travis' servers. I hope this explanation helps :)